<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>770</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1170</width>
    <height>770</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Настройки тем</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_but">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QListWidget" name="unkown_tema">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="move_tema_but">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_tema_but">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Нераспознанные темы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Официальные темы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tema_add_But">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить тему</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tema_add_line">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="official_tems">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="not_official_tems">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Неофициальные темы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_tema_but">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить тему</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ignore_tems">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Игнорируемые темы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ignore_tema_but">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Игнорировать</string>
    </property>
   </widget>
   <widget class="QListWidget" name="names_widget">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ignore_name_wg">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>250</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>210</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Игнорируемые имена</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Имена сотрудников</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ignore_name_but">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Игнорировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="not_ignor_tema_but">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="not_ignore_name_but">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Помиловать</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="wd_text_of_tema">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_text_but">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>700</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить текст</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="wd_text_of_tema_noname">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>475</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст темы в отчёте с именами</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>469</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст темы в отчёте без имён</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_names_but">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Редактировать имена</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
